plugins/dev-tools/package.json

```diff
@@ -53,10 +53,10 @@
   "devDependencies": {
     "@blockly/dev-scripts": "^3.1.1",
     "@types/dat.gui": "^0.7.5",
-    "blockly": "^10.0.0"
+    "blockly": "^10.2.0"
```

Comment:
Why did the minimum version need to be bumped?

Reply:
In order to access the relatively new accessor method: workspace.getSvgGroup()

Reply:
@maribethb does a major bump in dev tools (caused by the bump to the min version of Blockly) mean we have to do cascading major bumps in other plugins?

If so, we can just merge this into the `rc/v11.0.0` branch instead since that's going to require major bumps anyway.

Reply:
Sorry I had thought I replied to this. Yes it does, so let's put this in rc/v11 please. This should be marked as a breaking change due to bumping the minimum version

Reply:
Thanks, I've added a "!" to the title now.

Reply:
Okay I've adapted the changes to the rc/v11.0.0 branch and set that as the merge destination. From this branch, the plugin is already depending on a more recent version of blockly, so I don't need to make any changes to that, so I don't think it's a breaking change anymore in this context and I removed the "!" again.

Reply:
I believe that's fine. We'll do the actual `!` when we bump everything to v11.